src/Command/UpdateCommand.php

```diff
@@ -24,12 +24,17 @@ public function __construct(DbalSchemaCommand $schemaCommand)
     protected function configure(): void
     {
         $this->setName('dbal:schema:update')
-            ->addOption('force', 'f', InputOption::VALUE_NONE);
+            ->addOption('force', 'f', InputOption::VALUE_NONE)
+            ->addOption('no-transactions', 't', InputOption::VALUE_NONE);
```

Comment:
Thanks! Since transactions don't work for Mysql/Mariadb (AFAICT), which are the most popular, could we change this to not use transactions by default?

And then have a `--use-transaction|-t` flag to enable a transaction?